spec/jobs/payment_providers/adyen/handle_event_job_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PaymentProviders::Adyen::HandleEventJob, type: :job do
+  subject(:handle_event_job) { described_class }
+
+  let(:adyen_service) { instance_double(PaymentProviders::AdyenService) }
+  let(:result) { BaseService::Result.new }
+  let(:organization) { create(:organization) }
+  let(:event_json) { "{}" }
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)